.builds/alpine.yml

```diff
@@ -17,3 +17,9 @@ tasks:
     cd hidapi
     make
     make DESTDIR=$PWD/root install
+    make clean
```

Comment:
not sure why do you need to clean right after building it

Reply:
I don't think that this is relevant to the Haiku issue. Also, I'm not the author of that commit. CC: @z3ntu

Reply:
I'll give you both write access to my repository for that particular reason, just in case that there's any small changes regarding the CI configurations that need to be addressed.

Reply:
Yes this was merged as part of #131 - not sure why it shows up here. I've rebased the branch so it shows up fine now.

@Youw But to answer your question, it's needed because in the second step (`build-manual`) the full build should be done so a `make clean` is needed in the previous step for the second step to build everything. In addition to that, the build actually fails if not cleaned because of different CFLAGS iirc